docs/topics/logging.rst

```diff
@@ -194,42 +194,33 @@ scrapy.utils.log module
 .. module:: scrapy.utils.log
    :synopsis: Logging utils
 
-.. function:: configure_logging(settings=None)
+.. autofunction:: configure_logging
 
-    This function initializes logging defaults for Scrapy.
+``configure_logging`` is automatically called when using Scrapy commands,
+but needs to be called explicitly when running custom scripts. In that
+case, its usage is not required but it's recommended.
```

Comment:
Not sure if intentional since the additional documentation at another level, but shouldn't these lines be indented so they are at the same level as the autodocs generated from configure_logging?